tests/qos/qos_sai_base.py

```diff
@@ -0,0 +1,828 @@
+
+        logger.info("Add ip[6]tables rule to drop BGP SYN Packet from peer so that we do not ACK back")
+        for ipVersion in ipVersions:
+            updateIptablesDropRule(duthost, state="present", **ipVersion)
+
+        yield
+
+        logger.info("Remove ip[6]tables rule to drop BGP SYN Packet from Peer")
+        for ipVersion in ipVersions:
+            updateIptablesDropRule(duthost, state="absent", **ipVersion)
+
+    @pytest.fixture(scope='class')
+    def stopServices(self, duthost, swapSyncd):
+        """
+            Stop services (lldp-syncs, lldpd, bgpd) on DUT host prior to test start
+
+            Args:
+                duthost (AnsibleHost): Device Under Test (DUT)
+                swapSyncd (Fxiture): swapSyncd fixture is required to run prior to stopping services
+
+            Returns:
+                None
+        """
+        def updateDockerService(host, docker="", action="", service=""):
+            """
+                Helper function to update docker services
+
+                Args:
+                    host (AnsibleHost): Ansible host that is running docker
+                    docker (str): docker container name
+                    action (str): action to apply to service running within docker
+                    service (str): service name running within docker
+
+                Returns:
+                    None
+            """
+            host.command(
+                "docker exec {docker} supervisorctl {action} {service}".format(
+                    docker=docker,
+                    action=action,
+                    service=service
+                )
+            )
+
+        services = [
+            {"docker": "lldp", "service": "lldp-syncd"},
+            {"docker": "lldp", "service": "lldpd"},
+            {"docker": "bgp",  "service": "bgpd"},
+        ]
+
+        logger.info("Stop lldp, lldp-syncd, and bgpd services")
+        for service in services:
+            updateDockerService(duthost, action="stop", **service)
+
+        yield
+
+        logger.info("Start lldp, lldp-syncd, and bgpd services")
+        for service in services:
+            updateDockerService(duthost, action="start", **service)
+
+    @pytest.fixture(scope='class', autouse=True)
+    def disablePacketAging(self, duthost, stopServices):
+        """
+            disable packet aging on DUT host
+
+            Args:
+                duthost (AnsibleHost): Device Under Test (DUT)
+                stopServices (Fxiture): stopServices fixture is required to run prior to disabling packet aging
+
+            Returns:
+                None
+        """
+        if isMellanoxDevice(duthost):
+            logger.info("Disable Mellanox packet aging")
+            duthost.copy(src="qos/files/mellanox/packets_aging.py", dest="/tmp")
+            duthost.command("docker cp /tmp/packets_aging.py syncd:")
+            duthost.command("docker exec dyncd -it python packets_aging.py disable")
```

Comment:
'syncd' spelling

Reply:
thanks for this catch. I did not test this on Mellanox device yet